### PR TITLE
Bm/prop functionality

### PR DESCRIPTION
**Pull request by AurumValian**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Testing - building tests
- [ ] Formatting - no new features
- [ ] Styling- no new features
### Detailed Description (Why is this change required? What problem does it solve? Any challenges that arose while implementing this feature? If so, how were the addressed?)
This PR allows the user to search for Pokemon and creates a card when complete! It saves the Pokemon along with the user display name and slot to local storage, but that currently is not being fetched yet. 
### Files modified:
- [ ] index.html
- [x] App
- [ ] CardContainer
- [ ] PokeCard
- [x] PokeForm
- [ ] Login
- [x] FormContainer
- [ ] index.js
- [ ] Other files:

### CODE REVIEW

**Review by nware1066 — APPROVED** [review]
nice job, now the user can start getting that team together!

File: stretch-auth/src/App/App.js [hunk 1]
```diff
@@ -77,6 +79,10 @@ class App extends Component {
       .catch(error => console.log(error));
   }
 
+  savePokemonToUser(pokemon, slot) {
+    localStorage.setItem(`${this.state.currentUser.displayName} ${slot}`, JSON.stringify(pokemon))
```
Comment (H-Bo214) [inline stretch-auth/src/App/App.js:83]: Good idea saving this to Local Storage.

File: stretch-auth/src/FormContainer/FormContainer.js [hunk 2]
```diff
@@ -8,20 +9,69 @@ class FormContainer extends Component {
...
+      slot5: {},
+      slot6: {}
+    }
+
+    this.showPokemon = this.showPokemon.bind(this);
+  }
 
+  showPokemon(selectedPokemon, pokeName, slot) {
+    const foundPokemon = this.props.allPokemon.find(pokemon => {
+      return pokemon.name === selectedPokemon.toLowerCase() || pokemon.id === Number(selectedPokemon)
```
Comment (H-Bo214) [inline stretch-auth/src/FormContainer/FormContainer.js:25]: I also like the ability to search by name or number.

**Review by H-Bo214 — APPROVED** [review]

File: stretch-auth/src/App/App.js (hunk 1 above)
Comment (jordy1611) [reply stretch-auth/src/App/App.js:83]: Yea good call only ever having to download once

**Review by jordy1611 — COMMENTED** [review]
Really good functionality!

**Pull request merged**